recipes/nexusformat/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "nexusformat" %}
+{% set version = "0.4.20" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "58023fe0913a6d8273afc971cdc64ec0fc909f5d7d39a7819df0674ca2108ec2"
+
+build:
+  number: 0
+  entry_points:
+    - nxstack=nexusformat.scripts.nxstack:main
+  noarch: "python"
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
+    - numpy
```

Comment:
Is numpy a host requirement if you aren't building against it? I think you can remove this

Reply:
Unfortunately it is:

https://github.com/nexpy/nexusformat/blob/e1e6689b1651b8b7217158528336b80d8f126113/setup.py#L1-L16

Not sure why though since it isn't actually used further on in the script AFAICT

Reply:
I don't have any experience with submitting a condo-forge application, so I may not understand the question. Numpy is certainly required in order to run the nexusformat API. If the problem is that it is specifically imported in `setup.py`, that may be a historic relic from a previous stage of its development. I have checked that the following two lines can be removed from `setup.py` without causing any problems with pytest:
```
import pkg_resources
pkg_resources.require('numpy')
```

Reply:
The conda recipes make a distinction between build- (`requirements.host`) and run-time (`requirements.run`) dependencies. The former are necessary to ensure `python setup.py install` can be executed, while the latter are required to ensure the installation itself can be run, and are therefore also used to run the test block.

@scopatz I don't mind attaching a patch that removes these two lines... let me know if this necessary.

Reply:
There is no need to patch it here but, if you have time, it's worth making a PR upstream so it can be removed in the next version.

If you do, `COPYING` should be added to the `MANIFEST.in` file so the licence file is included in the tarball on PyPI (technically a requirement of BSD-3-Clause).

Reply:
See https://github.com/nexpy/nexusformat/pull/104